examples/multi-sensor/kitti-multimodal-image.json

```diff
@@ -0,0 +1,263 @@
+{
+    "frames": [
+        {
+            "name": "image_2/000000",
+            "url": "http://localhost:8686/items/kitti/tracking/training/image_02/0001/000000.png",
```

Comment:
change the url to a public url. Check examples\kitti\kitti_list.json for reference.